tests/helpers.py

```diff
@@ -710,6 +723,24 @@ def get_cephfs_name():
     return result['items'][0].get('metadata').get('name')
 
 
+def pull_images(image_name):
+    """
```

Comment:
Leave an empty line after each section in the docstring:

```
    """
    Function to pull images on all nodes

    Args:
        image_name (str): Name of the container image to be pulled

    Returns:
        bool: True if deletion of image pull is successful

    """
```

Reply:
Done